src/components/Offers/New/form-components/JobTypeComponent.js

```diff
@@ -15,25 +15,29 @@ const JobTypeComponent = ({ disabled, errors, requestErrors, control }) => (
                 fullWidth
                 id="job_type"
                 select
-                label="Job Type"
+                label="Job Type *"
                 value={value ? value : ""}
                 onChange={onChange}
                 onBlur={onBlur}
-                variant="outlined"
                 disabled={disabled}
                 error={!!errors?.jobType || !!requestErrors.jobType}
                 helperText={
                     `${errors.jobType?.message || requestErrors.jobType?.message || ""}`
                 }
+                {...textFieldProps}
             >
-                {JobOptions.map(({ value, label }) => (
-                    <MenuItem
-                        key={value}
-                        value={value}
-                    >
-                        {label}
-                    </MenuItem>
-                ))}
+                {JobOptions
+                    .filter((option) => option.value !== null)
+                    .map(({ value, label }) => (
+                        <MenuItem
+                            data-testid="job-type-item"
```

Comment:
Do we really need this data testid?

Reply:
As we discussed on slack, that is needed because the objective is to test if we don't have null data-values.